bitcoin/src/sign_message.rs

```diff
@@ -111,12 +111,10 @@ mod message_signing {
         /// Creates a `MessageSignature` from a fixed-length array.
         pub fn from_byte_array(
             bytes: &[u8; 65],
-        ) -> Result<MessageSignature, MessageSignatureError> {
+        ) -> Result<MessageSignature, secp256k1::Error> {
```

Comment:
FYI, in the future it'd be better to just fold this change into the the first commit to not confuse reviewers because although the diff looks like this changed the signature, really you've *added* a completely new method from API perspective.

But DO NOT CHANGE it now unless another problem is found, I'm ACKing this PR.